tests/emukit/quadrature/test_rbf_quadrature_kernel.py

```diff
@@ -4,25 +4,87 @@
 
 import numpy as np
 import GPy
+import pytest
 
 from emukit.model_wrappers.gpy_quadrature_wrappers import RBFGPy
-from emukit.quadrature.kernels import QuadratureRBF
+from emukit.quadrature.kernels import QuadratureRBFnoMeasure, QuadratureRBFIsoGaussMeasure
+from emukit.quadrature.kernels.integration_measures import IsotropicGaussianMeasure
 
 
-def test_rbf_qkernel_shapes():
+@pytest.fixture
+def qrbf_no_measure():
     x1 = np.array([[-1, 1], [0, 0], [-2, 0.1]])
     x2 = np.array([[-1, 1], [0, 0], [-2, 0.1], [-3, 3]])
     M1 = x1.shape[0]
     M2 = x2.shape[0]
     D = x1.shape[1]
 
     # integral bounds
-    lb = -3
-    ub = 3
+    bounds = [(-1, 2), (-3, 3)]
 
     gpy_kernel = GPy.kern.RBF(input_dim=D)
     emukit_rbf = RBFGPy(gpy_kernel)
-    emukit_qrbf = QuadratureRBF(emukit_rbf, integral_bounds=D * [(lb, ub)])
+    emukit_qrbf = QuadratureRBFnoMeasure(emukit_rbf, integral_bounds=bounds)
+    return emukit_qrbf, x1, x2, M1, M2, D
+
+
+@pytest.fixture
+def qrbf_iso_gauss_measure():
+    x1 = np.array([[-1, 1], [0, 0], [-2, 0.1]])
+    x2 = np.array([[-1, 1], [0, 0], [-2, 0.1], [-3, 3]])
+    M1 = x1.shape[0]
+    M2 = x2.shape[0]
+    D = x1.shape[1]
+
+    gpy_kernel = GPy.kern.RBF(input_dim=D)
+    emukit_rbf = RBFGPy(gpy_kernel)
+    measure = IsotropicGaussianMeasure(mean=np.arange(D), variance=2.)
+    emukit_qrbf = QuadratureRBFIsoGaussMeasure(rbf_kernel=emukit_rbf, measure=measure)
+    return emukit_qrbf, x1, x2, M1, M2, D
+
+
+def test_rbf_qkernel_no_measure_shapes(qrbf_no_measure):
+    emukit_qrbf, x1, x2, M1, M2, D = qrbf_no_measure
+
+    # kernel shapes
+    assert emukit_qrbf.K(x1, x2).shape == (M1, M2)
+    assert emukit_qrbf.qK(x2).shape == (1, M2)
+    assert emukit_qrbf.Kq(x1).shape == (M1, 1)
+    assert isinstance(emukit_qrbf.qKq(), float)
+
+    # gradient shapes
+    assert emukit_qrbf.dKq_dx(x1).shape == (M1, D)
+    assert emukit_qrbf.dqK_dx(x2).shape == (D, M2)
+
+
+def test_rbf_qkernel_no_measure_qK(qrbf_no_measure):
+    emukit_qrbf, _, x2, _, _, _ = qrbf_no_measure
+    # to check the integral, we check if it lies in some confidence interval.
```

Comment:
Does the code that did it still exist? May be worth adding it to the repo, even if it isn't called, and refer to it here. In case this computation needs to be repeated for some reason

Reply:
I still have it. Where would I put it though?

Reply:
next to this test would be fine

Reply:
I added 2 files, one for the qkernels, ine for the warped GP integrals